scripts/installer/Dockerfile

```
@@ -1,8 +1,7 @@
-FROM quay.io/centos/centos:stream8
+FROM quay.io/centos/centos:stream9
 
-RUN dnf -y install https://yum.puppetlabs.com/puppet7/puppet7-release-el-8.noarch.rpm
+RUN dnf -y install https://yum.puppetlabs.com/puppet7/el/9/x86_64/puppet7-release-7.0.0-19.el9.noarch.rpm
```

```suggestion
RUN dnf -y install https://yum.puppetlabs.com/puppet7-release-el-9.noarch.rpm
```

at some point they fucked up the `puppetX` folders and never updated the release symlinks in there :woman_shrugging:

---

awesome